src/devtools/client/inspector/computed/components/ComputedProperties.tsx

```diff
@@ -33,20 +33,18 @@ function ComputedProperties(props: PropsFromRedux) {
 
   let dark = false;
   let allPropertiesHidden = true;
-  const renderedProperties = properties.map((property, index) => {
+  const renderedProperties = properties.map(property => {
     const hidden = isHidden(property, search, showBrowserStyles);
-    if (!hidden) {
-      dark = !dark;
-      allPropertiesHidden = false;
-    }
+    if (hidden) return null;
+    dark = !dark;
+    allPropertiesHidden = false;
     const isExpanded = expandedProperties.has(property.name);
     const toggleExpanded = () => setComputedPropertyExpanded(property.name, !isExpanded);
 
     return (
       <ComputedProperty
-        key={index}
+        key={property.name}
```

Comment:
is it possible to have duplicate properties? perhaps not for the computed properties

Reply:
No, the properties are generated from a Map that uses the `name` property as key.